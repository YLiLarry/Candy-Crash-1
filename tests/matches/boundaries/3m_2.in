3 ..
hint
swap 1 1 left
