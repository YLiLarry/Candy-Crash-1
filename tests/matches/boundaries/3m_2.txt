__0 __1 __2
__2 __0 __1
__0 __1 __2
