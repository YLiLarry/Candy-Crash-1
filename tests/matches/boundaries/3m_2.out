
__0 __1 __2 
__2 __0 __1 
__0 __1 __2 
>> hint: 1 0 right

__0 __1 __2 
__0 __2 __1 
__0 __1 __2 

___ __1 __2 
___ __2 __1 
___ __1 __2 

__3 __1 __2 
__2 __2 __1 
__1 __1 __2 
